sklearn/base.py

```diff
@@ -369,7 +369,8 @@ class RegressorMixin:
     _estimator_type = "regressor"
 
     def score(self, X, y, sample_weight=None):
-        """Returns the coefficient of determination R^2 of the prediction.
+        """
+        Return the coefficient of determination R^2 of the prediction.
```

Comment:
Return -> Returns

Reply:
@eickenberg Following the standards of `numpydoc`, looks like we have to avoid infinitive verbs in summaries in order to avoid the following error:
```
SS05: Summary must start with infinitive verb, not third person (e.g. use "Generate" instead of "Generates")
```